src/app.module.ts

```diff
@@ -6,7 +6,8 @@ import { ConfigModule } from './modules/config/config.module';
 import { AuthModule } from './modules/auth/auth.module';
 import { SubmissionModule } from './modules/submission-adaptor/submission.module';
 import { PassportModule } from '@nestjs/passport';
-import { resolve } from 'path';
+
+const configPath = process.env.CONFIG_PATH ? process.env.CONFIG_PATH : '/config/config.json';
```

Comment:
wouldn't the default be best in `/etc/reviewer/..' ?

Reply:
ok will modify and also in other PR